mangaki/mangaki/management/commands/dedupe.py

```diff
@@ -0,0 +1,113 @@
+from django.core.management.base import BaseCommand, CommandError
+
+import os
+import csv
+from dedupe import Dedupe, consoleLabel, canonicalize
+from mangaki.models import Work
+
+
+def dedupe_training(category):
+    assert (category in ['mangas', 'animes']),"Only mangas or animes needs training"
+    data = {}
+    for work in Work.objects.filter(category__slug=category[:len(category)-1]):
+        data[work.id] = {'title': work.title, 'vo_title': work.vo_title}
+        for field in ['title', 'vo_title']:
+            if not data[work.id][field]:
+                data[work.id][field] = None
+    fields = [
+        {'field': 'title', 'type': 'String'},
+        {'field': 'vo_title', 'type': 'String'},
+    ]
+    output_file = 'dedupe/'+category+'_output.csv'
+    settings_file = 'dedupe/'+category+'_learned_settings'
+    training_file = 'dedupe/'+category+'_training.json'
+
+    deduper = Dedupe(fields)
+    deduper.sample(data)
+    consoleLabel(deduper)
+
+    if os.path.exists(training_file):
+        print('reading labeled examples from ', training_file)
+        with open(training_file, 'rb') as f:
+            deduper.readTraining(f)
+
+    deduper.train()
+
+    with open(training_file, 'w') as tf:
+        deduper.writeTraining(tf)
+
+    with open(settings_file, 'wb') as sf:
+        deduper.writeSettings(sf)
+
+    threshold = deduper.threshold(data, recall_weight=2)
```

Comment:
What is `recall_weight` ? What is its purpose ? What it is fixed to `2` ?
(add a comment to explain, maybe a `FIXME` to make it a configurable constant.)